ast/expressions.go

```diff
@@ -443,6 +444,47 @@ func (n *CompareSubqueryExpr) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+// ObjectNameExpr represents a table-level name expression, such as sequence/table/view etc.
+type ObjectNameExpr struct {
+	exprNode
+
+	// Name is the referenced object name expression.
+	Name *TableName
+}
+
+// Restore implements Node interface.
+func (n *ObjectNameExpr) Restore(ctx *RestoreCtx) error {
+	if err := n.Name.Restore(ctx); err != nil {
+		return errors.Trace(err)
+	}
+	return nil
+}
+
+// Format the ExprNode into a Writer.
+func (n *ObjectNameExpr) Format(w io.Writer) {
+	result := n.Name.Name.L
+	if n.Name.Schema.L != "" {
+		result = n.Name.Schema.L + "." + result
+	}
+	name := strings.Replace(result, ".", "`.`", -1)
+	fmt.Fprintf(w, "`%s`", name)
+}
```

Comment:
Could we to avoid using strings.Replace()?
```go
dbName, tbName := n.Name.Schema.L, n.Name.Name.L
if dbName == "" {
	fmt.Fprintf(w, "`%s`", tbName)
} else {
	fmt.Fprintf(w, "`%s`.`%s`", dbName, tbName)
}
```

Reply:
OK, addressed.